cypress-io/cypress
# feat: sync up protocol manager on after test

### Additional details


Wait on outstanding protocol events on after test

### Steps to test


Run protocol against the kitchen sink and ensure there aren't duplicate "about:blank" full snapshots

### How has the user experience changed?


n/a

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
